specification.md

```diff
@@ -407,7 +407,7 @@ The first way would be to use both "input", and "output":
 {
   "stateDataFilter": {
     "input": "${ {vegetables: .vegetables} }",
-    "output": "${ {vegetables: .vegetables[] | select(.veggieLike == true)} }"
+    "output": "${ {vegetables: [.vegetables[] | select(.veggieLike == true)}] }"
```

Comment:
typo here, should be 
         {vegetables: [ .vegetables[] | select(.veggieLike == true)]}

Reply:
fixed